vignettes/estimate_infections_workflow.Rmd

```diff
@@ -291,6 +291,10 @@ def <- estimate_infections(
   delays = delay_opts(delay),
   rt = rt_opts(prior = rt_prior)
 )
+#> Warning: There were 4 divergent transitions after warmup. See
+#> https://mc-stan.org/misc/warnings.html#divergent-transitions-after-warmup
+#> to find out why this is a problem and how to eliminate them.
+#> Warning: Examine the pairs() plot to diagnose sampling problems
```

Comment:
Nothing's changed except for the increase in divergences.

Reply:
Yes, as this will change again before the next release anyway.